src/tools/annotation/EllipticalRoiTool.js

```diff
@@ -249,10 +253,12 @@ export default class EllipticalRoiTool extends BaseAnnotationTool {
           'pixel',
           data.handles.initialRotation
         );
-        drawHandles(context, eventData, data.handles, handleOptions);
+        if (drawEllipseHandles) {
+          drawHandles(context, eventData, data.handles, handleOptions);
+        }
 
         // Update textbox stats
-        if (data.invalidated === true) {
+        if (data.invalidated === true && displayStats) {
```

Comment:
Can you give some more information as for what the changes are here and at line 372?

I am wondering why not follow a similar paradigm as https://github.com/videahealth/cornerstoneTools/blob/master/src/tools/annotation/RectangleRoiTool.js#L268 which determines whether to show the linked textbox or not.

Your approach may be better, though it would be good to understand how, since we may have more tools like this later :)

Reply:
I wanted to avoid recalculating the textbox logic each time we re-render. But I agree that the way it's currently being done for other tools is easier to understand so I will follow the existing logic.